### DISCUSSION TITLE
obj file missing

### DISCUSSION THREAD

**Discussion by qng-ahtg** [curl/curl#14679] [Q&A] [answered]
我配置好库的包含目录和库目录后，运行代码发现缺少obj文件。求各位看到我问题的大佬帮忙解答下这个问题。

**Comment by bagder** [comment]
(please use English)

> (translation by Google translate:) After I configured the include directory and library directory of the library, I found that the obj file was missing when I ran the code. I would like to ask the experts who saw my question to help me solve this problem.

Which curl version? How did you configure? What's the exact error?

**Reply by qng-ahtg** [reply]
My cloned GitHub code is in my project, the compiler is Visual Studio 2019. I set the directory and library directory as "Curl-8.9.1 \ Include \ Curl \", and the linker is set to "CURL \ Lib \". I encountered an error after running the project: Fatal Error LNK1104: Can't open the file "Project Path \ CURL-8.9.1 \ Lib.obj". Can this problem be solved?

**Reply by jay** [reply, marked as answer]
We need to know how you built curl to help but it sounds like the lib path is wrong. Maybe this will help: [Building your own application with libcurl (Visual Studio example)](https://github.com/curl/curl/blob/master/winbuild/README.md#building-your-own-application-with-libcurl-visual-studio-example)
